server.js

```diff
@@ -0,0 +1,18 @@
+
+var express = require('express'),
+  port = process.env.PORT || 3000,
+  app = express();
+
+app.use(express.static('./'));
+app.get('projects.html', function(){
+  response.json({message: 'what is happening'});
```

Comment:
You can remove this line and add the following to get the server to load up the index.html page:

```   console.log('New request:', request.url);
  response.sendFile('index.html', { root: '.' }); ```

Be sure to add all sections you want to hide/show in the index.html file.